### DISCUSSION TITLE
RedwoodJS runtime environment variables

### DISCUSSION THREAD

**Discussion by csellis** [vercel/vercel#5045] [Ideas] [answered]
Hi all! :wave: 

I've been streaming on RedwoodJS and was excited to see I could deploy to Vercel.

NextJS allows using environment variables at runtime by prepending NEXT_BUILD_ on the variable. Are we able to add something similar to the Redwood runtime?

I've tried digging through the PRs but I don't really know where to look.

Thanks!
Chris

**Comment by styfle** [comment, marked as answer]
I think you're looking for `REDWOOD_ENV_` prefix as seen on the redwood docs:

https://redwoodjs.com/docs/environment-variables#web

Vercel doesn't do anything special here so you'll want to use the redwood docs for further information.

**Comment by csellis** [comment]
Thanks Steven, It ended up being something undocumented in Redwood. It's
requiring the REDWOOD_ENV_ and the other variable. Worked out a solution
with David Price on the Redwood Team. Thank you!
